components/hab/src/command/studio/enter.rs

```diff
@@ -75,9 +74,6 @@ pub fn start(ui: &mut UI, args: &[OsString]) -> Result<()> {
                             config.auth_token,
                             Sensitivity::NoPrintValue);
     set_env_var_from_config(BLDR_URL_ENVVAR, config.bldr_url, Sensitivity::PrintValue);
-    set_env_var_from_config(CTL_SECRET_ENVVAR,
```

Comment:
It would be good to leave a comment here saying that we are intentionally leaving one of the `cli.toml` fields out, along with why.

It may also be useful to print a message to the user if they have a `CTL_SECRET` set in their environment or `cli.toml` that we are explicitly ignoring it, as well as an explanation why.

(of course, the second suggestion would also do the job of the first suggestion.)

Reply:
Good point! Message added.